src/less/blocks.less

```diff
@@ -63,4 +63,30 @@
     overflow: hidden;
     clip: rect(1px 1px 1px 1px);
     clip: rect(1px, 1px, 1px, 1px);
-}
\ No newline at end of file
+}
+
+.CodeMirror {
+    z-index: 0;
```

Comment:
Existing CM installations may require CM to have a different z-index. Do we need to force them to zero?

Reply:
I really couldn't find other way to not make CodeMirror over the modal except this way...

Reply:
OK, good enough for now. If we find a reason to revisit, maybe we can programmatically compute the z-index of the modal to be cm.zIndex+1. But let's leave that for now.